models/BaseEntity.cfc

```diff
@@ -1041,8 +1057,10 @@ component accessors="true" {
 			}
 		);
 		activateGlobalScopes();
-		var data = retrieveQuery()
-			.from( tableName() )
+
+		var q = variables._hasParentEntity ? newQuery() : retrieveQuery();
```

Comment:
I'm not sure this is correct.  Calling `newQuery` here will remove any constraints already added to the query, e.g. `user.where( "active", 1 ).orderBy( "createdDate" ).find( 1 )` will lose the `where` and `orderBy`.

Reply:
I probably need to double back on this, then, as `newQuery` was where I could place all of the join logic for parents and children without having to duplicate it all over.  `retreiveQuery`  can't be used because it affects the subqueries.  Currently, I don't use `newQuery` for parent/child entities, then the joins for the parent attributes don't happen.